test/plugin/scenarios/elasticsearch-rest-high-level-6.x-scenario/config/expectedData.yaml

```diff
@@ -307,124 +307,6 @@ segmentItems:
       - {key: db.type, value: Elasticsearch}
       - {key: db.instance, value: not null}
       skipAnalysis: 'false'
-    - operationName: Elasticsearch/IndexRequest
```

Comment:
Why do these get removed?

Reply:
Moved to elasticsearch-transport-6.x-scenario

Reply:
The version of rest-high-level-client changes too much, cannot be cover with a test case, in order not to affect the test of transport, so there are two cases.

Reply:
It is fine you don't add more versions to support.
Please update doc according to the latest status.

Reply:
Supported-list.md updated.